network-api/networkapi/wagtailpages/templates/wagtailpages/tags/cta.html

```diff
@@ -4,6 +4,6 @@
 	{% include './cta/petition.html' %}
 {% endif %}
 
-{% if cta_type == 'Signup' %}
+{% if page.signup %}
```

Comment:
Why did this need to change? My instinct is that this and line 3 should be of the same format (though I'm not sure which.)

Reply:
Yeah, I agree, but for some reason it doesn't pull in `signup.html` when using `cta_type  == 'Signup'`. @Pomax would you happen to know why that is? I looked in the codebase and it appears that 'signup' is a `cta_type`. 🤷‍♀

Reply:
`cta_type` comes from https://github.com/mozilla/foundation.mozilla.org/blob/master/network-api/networkapi/wagtailpages/templatetags/mini_site_tags.py#L19-L34, which needs to be loaded on the page by using `{% load mini_site_tags %}`, because the signup used on _only_ be for minisites so we never moved the cta tag out of that set of tags, and then you need to make sure to use the `cta` tag itself in the template.